src/compounds/legacy-product-table/src/index.tsx

```diff
@@ -562,6 +562,47 @@ const Eligibility: React.FC<EligibilityProps> = ({
   )
 }
 
+interface TelephoneInfoProps {
+  telephone?: string
+}
+
+const TelephoneInfo: React.FC<TelephoneInfoProps> = ({ telephone }) => {
+  const {
+    theme: { name, colors = {}, fontWeights = {}, fonts = {}, space = {} }
+  }: any = useThemeUI()
+
+  const iconColor = name === 'Money' ? colors.fuschia : colors.primary
+
+  return (
+    <div
+      sx={{
+        mr: '10px',
+        fontWeight: fontWeights.xbold,
+        fontFamily: fonts.body,
```

Comment:
```suggestion
        fontFamily: 'xbold',
        fontFamily: 'body',
```

Might not need to use `useThemeUI`